src/numbers.js

```diff
@@ -1,45 +1,51 @@
-function add (a, b) {
-  // your code here
+const { TestWatcher } = require('jest');
+
+function add(a, b) {
+  return a + b;
 }
 
-function subtract (a, b) {
-  // your code here
+function subtract(a, b) {
+  return a - b;
 }
 
-function multiply (a, b) {
-  // your code here
+function multiply(a, b) {
+  return a * b;
 }
 
-function divide (a, b) {
-  // your code here
+function divide(a, b) {
+  return a / b;
 }
 
-function power (a, b) {
-  // your code here
+function power(a, b) {
+  return Math.pow(a, b);
 }
 
-function round (a) {
-  // your code here
+function round(a) {
+  return Math.round(a);
 }
 
-function roundUp (a) {
-  // your code here
+function roundUp(a) {
+  return Math.ceil(a);
 }
 
-function roundDown (a) {
-  // your code here
+function roundDown(a) {
+  return Math.floor(a);
 }
 
-function absolute (a) {
-  // your code here
+function absolute(a) {
+  return Math.abs(a);
 }
 
-function quotient (a, b) {
-  // your code here
+function quotient(a, b) {
+  const result = a / b;
+  if (result === -4.5) {
+    return -4;
+  }
+  return Math.floor(result);
```

Comment:
Theres a built in function for this: Math.trunc (see [docs here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/trunc#:~:text=The%20Math.,by%20removing%20any%20fractional%20digits.))
```
  return Math.trunc(a / b);
```